### PR TITLE
Make default value of enable-hns flag true

### PR DESCRIPTION
**Pull request by Tulsishah**
### Description
- Make default value of `enable-hns` flag true.
- Remove enableEmptyManagedFolder flag true with enableHNS flag true as it is changing flag value for flat bucket also.
- Rename enableEmptyManagedFolder in dir.go with IncludeFoldersAsPrefixes.
- Removed/Updated all the unit tests for changes.

### Link to the issue in case of a bug fix.
NA

### Testing details
1. Manual - NA
2. Unit tests - NA
3. Integration tests - Automated

### CODE REVIEW

File: cfg/config.go
Comment (kislaykishore) [inline cfg/config.go:1]: Please don't edit this file manually. The params.yaml should be changed to reflect the correct default value. Post that, run `make build` to generate this configuration.

**Review by kislaykishore — CHANGES_REQUESTED** [review]

File: cfg/config.go
Comment (Tulsishah) [reply cfg/config.go:1]: Thanks.

**Review by kislaykishore — APPROVED** [review]
Approving for config changes. Please get the business logic change reviewed from someone in the know.

**Review by ankitaluthra1 — APPROVED** [review]

File: cfg/rationalize.go [hunk 1]
```diff
@@ -27,11 +27,6 @@ func decodeURL(u string) (string, error) {
 
 // Rationalize updates the config fields based on the values of other fields.
 func Rationalize(c *Config) error {
-	// The EnableEmptyManagedFolders flag must be set to true to enforce folder prefixes for Hierarchical buckets.
-	if c.EnableHns {
-		c.List.EnableEmptyManagedFolders = true
```
Comment (ashmeenkaur) [inline cfg/rationalize.go:32]: Default value of EnableEmptyManagedFolders should also have been changed to true? @Tulsishah

Comment (Tulsishah) [reply cfg/rationalize.go:32]: As discussed offline it is intended.

**Pull request merged**